home/init.asm

```diff
@@ -94,13 +94,11 @@ Init:: ; 17d
 
 ; Initialize the RNG state. It can be initialized to anything but zero; this is just a simple way of doing it.
 	ld hl, wRNGState
-	ld a, "R"
+	ld a, 1
```

Comment:
Is this necessary? IMO just leave the RNG state as-is, it's only a few bytes more, and I'd rather not change its behavior.